pkg/network/cloud_network.go

```diff
@@ -38,18 +41,27 @@ func renderCloudNetworkConfigController(conf *operv1.NetworkSpec, cloudBootstrap
 	data.Data["PlatformTypeAzure"] = v1.AzurePlatformType
 	data.Data["PlatformTypeGCP"] = v1.GCPPlatformType
 	data.Data["CloudNetworkConfigControllerImage"] = os.Getenv("CLOUD_NETWORK_CONFIG_CONTROLLER_IMAGE")
-	data.Data["KubernetesServiceHost"] = cloudBootstrapResult.APIServers[bootstrap.APIServerDefault].Host
-	data.Data["KubernetesServicePort"] = cloudBootstrapResult.APIServers[bootstrap.APIServerDefault].Port
+	data.Data["KubernetesServiceHost"] = cloudBootstrapResult.APIServers[bootstrap.APIServerDefaultLocal].Host
```

Comment:
@kyrtapz  I forgot the difference between APIServerDefaultLocal and APIServerDefault, do we need to distinguish it for self-hosted and managed?

Reply:
this variable should do exactly that: https://github.com/openshift/cluster-network-operator/blob/3b3909bd4f64390fc2b0c80b1be92ff6370209a5/pkg/bootstrap/types.go#L105-L108

Reply:
Yes, that is correct